extensions/amp-ad-exit/0.1/config.js

```diff
@@ -165,10 +167,10 @@ function assertTarget(name, target, config) {
  * @return {string} The vendor's iframe URL
  */
 export function assertVendor(vendor) {
-  user().assert(ANALYTICS_CONFIG &&
-      ANALYTICS_CONFIG[vendor] &&
-      ANALYTICS_CONFIG[vendor]['transport'] &&
-      ANALYTICS_CONFIG[vendor]['transport']['iframe'],
+  user().assert(ANALYTICS_IFRAME_TRANSPORT_CONFIG &&
+      ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor] &&
+      ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor]['transport'] &&
+      ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor]['transport']['iframe'],
   'Unknown vendor: ' + vendor);
-  return ANALYTICS_CONFIG[vendor]['transport']['iframe'];
+  return ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor]['transport']['iframe'];
```

Comment:
I think you can just do return user().assertString(ANALYTICS_IFRAME_TRANSPORT_CONFIG[vendor] && ...);

Reply:
Done